README.md

```diff
@@ -17,7 +17,7 @@ A distributed ledger for ecology built on top of the
 
 ## Getting Started
 
-See [Getting Started](docs/getting_started.md).
+See [Getting Started](https://github.com/regen-network/regen-ledger/tree/master/docs/src/getting-started).
```

Comment:
let's use relative links
```suggestion
See [Getting Started](./docs/src/getting-started/README.md).
```

Reply:
Thanks @gotjoshua for the PR, please note that the `docs` folder structure changed a little bit after #158